mesh/amqp.c

```diff
@@ -1172,3 +1175,21 @@ bool mesh_amqp_is_ready(struct mesh_amqp *amqp)
 {
 	return amqp->amqp_state == MESH_AMQP_STATE_CONNECTED;
 }
+
+bool mesh_amqp_opcodes_whitelist_add(struct mesh_amqp *amqp, uint32_t opcode)
+{
+	uint32_t *_opcode = l_new(uint32_t, 1);
+	*_opcode = opcode;
+
+	return l_queue_push_tail(amqp->opcodes_whitelist, _opcode);
```

Comment:
I think you could avoid `l_new` by storing opcodes directly, via `L_UINT_TO_PTR`/`L_PTR_TO_UINT`, see model.c mod->subs for an example.